jms/src/main/scala/akka/stream/alpakka/jms/JmsConnector.scala

```diff
@@ -40,11 +40,23 @@ private[jms] trait JmsConnector { this: GraphStageLogic =>
       onSessionOpened(session)
     }
 
-  private[jms] def initSessionAsync(dispatcher: Dispatcher): Future[Unit] = {
-    ec = materializer match {
+  private[jms] def executionContext(attributes: Attributes): ExecutionContext = {
```

Comment:
This and a few other things in this class should be just regular `private` or `protected`.

Reply:
agreed; I wanted to follow the style of the existing implementation, I'll change that to simpler visibility restrictions.